ibis/pandas/execution/tests/test_operations.py

```diff
@@ -34,40 +34,6 @@ def test_read_with_undiscoverable_type(client):
         client.table('df')
 
 
-merge_asof_minversion = pytest.mark.skipif(
-    pd.__version__ < '0.19.2',
```

Comment:
should prob bump the min of pandas to 0.20.3 in any event

Reply:
This was just duplicated code. Can bump in another issue at some point, but it's fine to leave at 0.19.2. If lots of things stop working or some core piece of software in ibis absolutely needs pandas >=0.19.2 then we can update.